install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Another unstated goal was that everything would be a single file, which keeps the installation easy. I could see adding all the alias stuff to an install target in tldr itself, (tldr -i).